README.rst

```diff
@@ -572,15 +572,15 @@ Following is the description of the configuration:
 Adding a model link
 ~~~~~~~~~~~~~~~~~~~
 
-If you want to add a link that contains url of ``add`` or ``list`` page of a model
-then you can use following syntax. Users will only be able to see links for
+To add a link that contains url of ``add`` or ``list`` page of a model
```

Comment:
url > URL.

BTW add and list are not code samples so I don't think we should highlight them as such, I belive  we can say add form and change list page